pkgs/development/compilers/adoptopenjdk-bin/generate-sources.py

```diff
@@ -32,8 +32,9 @@ def generate_sources(release, assets):
         if asset["openjdk_impl"] not in impls: continue
         if asset["heap_size"] != "normal": continue
         if asset["architecture"] not in arch_to_nixos: continue
+        print(asset)
 
-        version, build = RE_RELEASE_NAME.match(asset["release_name"]).groups()
+        version, build = asset["version_data"]["semver"].split("+")
```

Comment:
I'd keep the comment mentioning "jdk-11.0.1+13" as an example name, makes it easier to understand what is going on here.